### DISCUSSION TITLE
installation

### DISCUSSION THREAD

**Discussion by jd1701d** [ParamITNL/fs22-AnimalHelperMod#21] [Q&A]
I'm running fs22 on windows. To install this mod is it just drag and drop it into the mods folder or do I need to do something special?
When I put it into my mods folder it won't show up in my game or anything.

**Comment by Do3torWumbo** [comment]
Did you ever figure this out im having the same issue

**Reply by Cjarsons** [reply]
inside the zip there should be only be the files and folders you see on code section, if there are folders above them that's the issue
